### PR TITLE
Facehugger inhands, xeno tweaks, and monkey bugfixes

### LINKED ISSUE #4735: Monkeys can uncuff people while cuffed themselves
They might also be able to strip while cuffed.

I was also in a chair at the time

### PR DESCRIPTION
**Pull request by phil235**
Some of these changes are salvaged from #8629
- Aliens now use the same examine as monkeys. Monkey and aliens now all use carbon/examine(). (relevant to #3574)
- Fixes monkeys being able to strip and uncuff people while handcuffed themselves. Fixes #4735
- Add an inhand sprite for facehuggers (taken from Xhuis) (but they don't appear for large queens, it'd look ugly).
- Xenos now regenerate plasma on alien weed even when injured but at half the normal rate. They also heal their clone damage now.
- Moved queen/large/update_icons() to humanoid/update_icons.dm
- Alien humanoids and monkeys can now use "me".
- Alien hivemind chat now uses <span class='alien'> (dark purple). Also the alien queen name now appears bigger in the hivemind chat.

### CODE REVIEW

File: code/modules/mob/living/carbon/alien/organs.dm [hunk 1]
```diff
@@ -79,6 +79,7 @@
 			var/mod = 1
 			if(!isalien(owner))
 				mod = 0.2
+			owner.adjustPlasma(plasma_rate/2)
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/alien/organs.dm]: ```*0.5``` instead please, same effect, slightly cheaper.

**Comment by WJohn** [comment]
:+1: x :100:  Thank you thank you thank you.

**Comment by daman997** [comment]
/me puts on his suit and xeno mask

**Comment by bgobandit** [comment]
"Alien humanoids and monkeys can now use "me"."

are you sure about this

do you know what you are unleashing

(aliens getting around the whisper power, that's what. I bet you were thinking about xeno ERP you pervert)

**Comment by WJohn** [comment]
Shhh ignore those two nothing could POSSIBLY go wrong.

Merge this please.

**Comment by melandor0** [comment]
Aliens used to have /me

Amy Lessen.

**Comment by cyfause** [comment]
amy time

**Pull request merged**